### DISCUSSION TITLE
Don't fade out the scroll bar on collections settings/data-model

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13643] [Feature Requests]
In terms of usability, a scroll bar that just appears when hovering over the area is awkward. Where is the scroll bar if you don't know it appears when you hover.

How about leaving it permanently showing (if needed) so people know what to grab and where to grab it.

**Comment by benhaynes** [comment]
Should be possible in a CSS override. :)

**Comment by rijkvanzanten** [comment]
And this is also very much a macOS painpoint 😁 

![CleanShot 2022-05-31 at 12 42 20@2x](https://user-images.githubusercontent.com/9141017/171228459-dfdea6b8-3b78-4fb7-bcc2-8d4702aed802.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
